components/collection/drop/modal/PaidMint.vue

```diff
@@ -137,8 +129,7 @@ const moveSuccessfulDrop = computed<boolean>(
   () =>
     imagePreloadingCompleted.value &&
     Boolean(mintingSession.value.items.length) &&
-    Boolean(mintingSession.value.txHash) &&
-    props.action.details.status === TransactionStatus.Finalized,
```

Comment:
Is it safe to delete this condition? What if the TX failed?

Reply:
> This is the line on which I need feedback. Sometimes, the status was never changed to `Finalized`

you mean the transaction was successful, but status was never updated to `Finalized` ?

Reply:
> If possible, there are two TX with the same SN would be conflicted

Oh, this one is easier. Let me check

> you mean the transaction was successful, but status was never updated to Finalized ?

It seems like successful because we can safely close the tabs, and later on, the item will be updated

Reply:
So far, it looks good if the transaction fails. It will blocked to the next step

<img width="2168" alt="Screenshot 2024-05-29 at 19 22 16" src="https://github.com/kodadot/nft-gallery/assets/734428/b14dba70-35ff-4e68-8fa9-141a233dd184">